packages/new-polymath-ui/src/components/ModalTransactionQueue/TransactionItem/TransactionItem.tsx

```diff
@@ -78,3 +81,9 @@ export const TransactionItem = ({ transaction }: TransactionItemProps) => {
     </sc.Wrapper>
   );
 };
+
+TransactionItem.defaultProps = {
+  getTitle: getTransactionTitle,
+};
```

Comment:
so we really need to have the defaultProps defined 2 times? The one inlined in the function declaration is really bloating readability :(

Reply:
The one inlined is not a definition, it's a type declaration. We could probably refactor it so that the type is defined separately for better readability, but it is necessary for it to be typesafe

Reply:
Changed it to be more readable